### PR TITLE
Jz explorethedata styling

### PR DESCRIPTION
**Pull request by wetchler**

### CODE REVIEW

File: page-data.php [hunk 1]
```diff
@@ -23,6 +23,7 @@
     ?>
 
 <div id="js-TJIChartView" class="row">
+	<h2>Current Data Set: <span class="tji-current-dataset"></span></h2>
```
Comment (wetchler) [inline page-data.php]: I think we can remove this line. Currently, TJIChartView obliterates everything inside the `js-TJIChartView` div once the data loads. TJIChartView also already generates an element displaying the number of records.

File: page-data.php [hunk 2]
```diff
@@ -51,8 +52,9 @@
       ],
       '#js-TJIChartView',  // Element to build the charts into
       '#secondary',  // Element to put the filter panel inside
-      '<div class="col-sm-12 col-lg-6" />',  // Wrapper for chart canvases
-      '<div class="col-sm-12 record-count">{count} records</div>',  // Template for record count element
+      '<div class="col-xs-12 col-sm-12 col-md-6 col-xlg-4 tji-chart" />',  // Wrapper for chart canvases
+      '<div class="col-sm-12 tji-data-count">{count} records</div>'  // Template for record count element
```
Comment (wetchler) [inline page-data.php]: Nit: Can we keep this classname consistent by calling it `tji-record-count`?

File: sass/tji/_tji.scss [hunk 3]
```diff
@@ -13,7 +13,11 @@
...
 
+/*--------------------------------------------------------------
+## Explore the Data Styles
+--------------------------------------------------------------*/
+@import "data";
 /*--------------------------------------------------------------
 ## Helpers
 --------------------------------------------------------------*/
-@import "helpers";
+@import "helpers";
```
Comment (wetchler) [inline sass/tji/_tji.scss]: Can we just take all the stuff in `helpers` and put it in `data`? Then delete `helpers`? I don't see much of a need for two separate files....

File: sass/tji/_data.scss [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+  border: 1px solid #ebebeb;
+  transition: all 0.25s;
+  
+  &:hover {
+    box-shadow: 0px 0px 20px rgba(64,64,64,0.15);
+    background-color: #fff;	    
+  }
+  
+  @media (min-width: 1500px) {
+	  -ms-flex-preferred-size: 33.33%;
```
Comment (ifsimicoded) [inline sass/tji/_data.scss]: We have flexbox classes for this sort of thing and overwriting library styles likes this is a tricky animal and tends to get confusing if there's a lot of it. So two potential other options so we don't have to go down this path, 
1) we can use https://github.com/hugeinc/flexboxgrid-sass, which is a sass-ified version of flexboxgrid and set our breakpoints so large could be 1500.
2) do we need to worry about people with screens that are larger than 1500? Is that all that common?

File: page-data.php [hunk 5]
```diff
@@ -51,8 +52,9 @@
       ],
       '#js-TJIChartView',  // Element to build the charts into
       '#secondary',  // Element to put the filter panel inside
-      '<div class="col-sm-12 col-lg-6" />',  // Wrapper for chart canvases
-      '<div class="col-sm-12 record-count">{count} records</div>',  // Template for record count element
+      '<div class="col-xs-12 col-sm-12 col-md-6 col-xlg-4 tji-chart" />',  // Wrapper for chart canvases
```
Comment (ifsimicoded) [inline page-data.php]: I don't think 'col-xlg-4' is a thing -- can we remove this so it's not confusing.

File: sass/tji/_data.scss [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+----------------------------------------------------------------
+# GENERAL
+# CHARTS
+
+/*--------------------------------------------------------------
+## CHARTS
+--------------------------------------------------------------*/
+
+#js-TJIChartView {
+  width: 100%;
```
Comment (ifsimicoded) [inline sass/tji/_data.scss]: Adding a width and margin-left here is fighting-with/overwriting how the flexboxgrid .row class is being used -- and is causing the chartview wrapper to expand past the content_area/primary container. This approach is brittle and adds complexity to our layout that is difficult. Could we find another way to achieve desired spacing? Do we need these two styles?

File: sass/tji/_data.scss [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+  padding: 2em;
+  background-color: #fbfbfb;
+  box-shadow: 2px 3px 6px rgba(64,64,64,0.5);
```
Comment (ifsimicoded) [inline sass/tji/_data.scss]: This is a cool effect. Usually this type of shadow is used for visual separation, but if the entire chart area is raised, I'm not sure what the separation is for. Would this make more sense applied to individual charts instead?

File: sass/tji/_data.scss [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+
+  border: 1px solid #ebebeb;
+  transition: all 0.25s;
+  
+  &:hover {
+    box-shadow: 0px 0px 20px rgba(64,64,64,0.15);
+    background-color: #fff;	    
+  }
+  
+  @media (min-width: 1500px) {
```
[the remark this answers is not in the document]
Comment (ifsimicoded) [reply sass/tji/_data.scss]: In tji/_general.scss we're using ems, pixels, min-widths and max-widths, so it seems like we could really use some consistency implemented.

**Comment by LayaTaal** [comment]
A version of the desktop styling with download button. I think it will be useful to describe what the user is downloading also.

explore_desktop

**Pull request closed without merging**